### ISSUE TITLE
Can we add emotions

### ISSUE THREAD

**Issue by sajal-madan-ds** [k2-fsa/ZipVoice#153]
Architecturally, can we add emotions to the ZipVoice non-dialogue model?

**Comment by zhu-han** [comment]
Yes. You'll need to add emotion as another condition in a proper way and train the model on a speech dataset with emotion labels.

**Comment by sajal-madan-ds** [comment]
Can I use the pre-trained model weights and then make these changes? @zhu-han 

Are you suggesting something like this? - https://github.com/RaduBolbo/F5-TTS-Emotional-CFG

**Comment by zhu-han** [comment]
Yeah, you can fine-tune the model with this condition.

**Comment by Alekksander66** [comment]
Hello everyone, hope you're doing well, may we have some more explanations on how to train a model from scratch and adding emotions ? Thank you !
